models/labels/addresses/infrastructure/persona/flashloans/labels_flashloans_ethereum.sql

```diff
@@ -0,0 +1,65 @@
+{{config(
+    alias='flashloans_ethereum',
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["niftytable"]\') }}'
+    )
+}}
+
+WITH basic_info AS (
+SELECT DISTINCT target                   AS address,
+                'infrastructure'         AS category,
+                'persona'                AS label_type,
+                'flashloan_users'        AS model_name,
+                'ethereum'               AS blockchain,
+                'Aave v2 Flashloan User' AS name
+FROM
+    {{ source('aave_v2_ethereum', 'LendingPool_evt_FlashLoan') }}
+WHERE
+    amount != 0
+
+UNION ALL
+SELECT DISTINCT target                   AS address,
+                'infrastructure'         AS category,
+                'persona'                AS label_type,
+                'flashloan_users'        AS model_name,
+                'ethereum'               AS blockchain,
+                'Aave v3 Flashloan User' AS name
+FROM
+    {{ source('aave_v3_ethereum', 'Pool_evt_FlashLoan') }}
+WHERE
+    amount != 0
+
+UNION ALL
+SELECT DISTINCT recipient                    AS address,
+                'infrastructure'             AS category,
+                'persona'                    AS label_type,
+                'flashloan_users'            AS model_name,
+                'ethereum'                   AS blockchain,
+                'Balancer v2 Flashloan User' AS name
+FROM
+    {{ source('balancer_v2_ethereum', 'Vault_evt_FlashLoan') }}
+WHERE
+    amount != 0
+
+UNION ALL
+SELECT DISTINCT recipient                   AS address,
+                'infrastructure'            AS category,
+                'persona'                   AS label_type,
+                'flashloan_users'           AS model_name,
+                'ethereum'                  AS blockchain,
+                'Uniswap v3 Flashloan User' AS name
+FROM
+    {{ source('uniswap_v3_ethereum', 'Pair_evt_Flash') }}
+WHERE
+    amount0 != 0
+)
+
+SELECT *
```

Comment:
@Hosuke it looks like it's column order in the select here is incorrect, which is then union'ed into the `labels_infrastructure` spell. in your [query](https://dune.com/queries/2355468), you can see values aren't in the right category. please ensure this spell follows this pattern:
- blockchain
- address
- name
- category
- contributor
- source
- created_at
- updated_at
- model_name
- label_type